src/util/Whamm.v3

```diff
@@ -43,24 +43,40 @@ type WhammArg {
 	case Local(t: ValueType, i: int);
 }
 
+type WhammPredicate {
```

Comment:
What about just making it a class, i.e. a wrapper around a `WhammParam.Call`, or even a the function and its args (i.e. not a `WhammParam`)?